manifests/redhat.pp

```diff
@@ -8,6 +8,7 @@
         # they don't exist so the service won't start.
         file { '/var/log/mariadb':
           ensure => 'directory',
+          mode   => '0755',
```

Comment:
Did you test this? Changing the mode from `0700` to `0755` does not grant write access, but only read access. So I'm wondering how this is supposed to fix #182. Maybe you could provide more insights.

Besides that it would be useful to know which version of MariaDB or MySQL (or ...) is being used.

Reply:
@fraenki i'd checked it again, with this mode the server is able to start and loggs correctly.
I've checked with
- MariaDB-Enterprise 10.6.14 / Galera Enterprise 4 - 26.4.14
- MariaDB-Enterprise 10.4.30 / Galera Enterprise 4 - 26.4.14

Reply:
Could you probably provide the output of the failed Puppet Run and the MariaDB log? And maybe your puppet-galera config so I can try to reproduce this.

Reply:
I've resurrected the acceptance tests and they run successfully on Rocky 8 and MariaDB 10.6:
https://github.com/markt-de/puppet-galera/actions/runs/5716432830/job/15488115817

So please provide the additional information (see previous comment) :)